tests/plugins/list/backspace.html

```diff
@@ -871,3 +871,53 @@
 		<li>3</li>
 	</ul>
 </textarea>
+
+<textarea id="delete_into_nested_list1">
+	<ul>
+		<li>a
+			<ul>
+				<li>^</li>
+			</ul>
+		</li>
+		<li>b</li>
+		<li>c</li>
+	</ul>
+	=>
+	<ul>
+		<li>a
+			<ul>
+				<li>^b</li>
+			</ul>
+		</li>
+		<li>c</li>
+	</ul>
+</textarea>
```

Comment:
^ is the caret position
and the => reflects the before/after
(in this case, pressing del)